README.md

```diff
@@ -1,6 +1,28 @@
 # cozmo-csharp-sdk
-Anki Cozmo CSharp Example Interface
+Anki Cozmo C# SDK Interface
 
-Anki will not be actively maintaining this project at this time.  The project is meant to show how a c# project could be built that interacts with Cozmo, but is not a complete SDK comparable to the actively maintained python repository.
+Unlike the Cozmo Python SDK - https://github.com/anki/cozmo-python-sdk - the C# SDK is not actively maintained by Anki, and is instead meant as starting point for anyone who wishes to build their own C# SDK for interacting with Cozmo.
 
-The included project can be built on osx via Xamarin studio. or windows via Visual Studio.  The project is a sample app which demonstrates how to send a specific action and register for specific callbacks.  This approach should scale to the other message types in clad DLL.  The app also provides bindings to package and unpackage tcp packets intercepted from the cozmo app to and from clad messages.
+We will release an updated CladCSharp.dll for each subsequent release of the Cozmo app, so that you don't need to worry about the binary format of the messages when they're sent over the network.
+
+The Cozmo Python SDK GitHub repo: https://github.com/anki/cozmo-python-sdk
+
+Learn more about Cozmo: https://anki.com/cozmo
+
+Learn more about the SDK: https://developer.anki.com/
+
+SDK documentation: http://cozmosdk.anki.com/docs/
+
+Forums: https://forums.anki.com/
+
+
+
+The included project can be built on macOS via Xamarin Studio, or on Windows via Visual Studio. This project is a very small sample application which demonstrates how to send an action to Cozmo, and how to register for specific callbacks to receive incoming messages.  Other messages can be found by browsing the CladCSharp.dll assembly, and can be send and listened for using a similar approach.
```

Comment:
typo in "can be send" - should be sent.
I think "sent and received" might read better than "sent and listened for" to?